### PR TITLE
Patch Not Only Just a Cannon

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Added Patch for Not Only Just a Cannon. The Sieges don't bring the correct ammo yet.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Not Only Just A Cannon/Patch_Cannon_Ammo.xml [hunk 1]
```diff
@@ -0,0 +1,831 @@
...
+			  <ThingDef Class="CombatExtended.AmmoDef" Name="Base164mmCannonShell" ParentName="BaseBullet" Abstract="true">
+				<graphicData>
+				  <shaderType>TransparentPostLight</shaderType>
+				</graphicData>
+				<projectile Class="CombatExtended.ProjectilePropertiesCE">
+				  <speed>100</speed>
+				  <soundExplode>MortarBomb_Explode</soundExplode>
+				  <flyOverhead>false</flyOverhead>
+				  <dropsCasings>true</dropsCasings>
+				  <casingMoteDefname>Mote_BigShell</casingMoteDefname>
```
Comment (N7Huntsman) [inline Patches/Not Only Just A Cannon/Patch_Cannon_Ammo.xml:65]: These shells probably shouldn't have a mote.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
There's also a "Dragonator" cannon that needs to be either patched or removed. Something should be done with it one way or another.

File: Patches/Not Only Just A Cannon/Patch_Cannon_Turret.xml [hunk 2]
```diff
@@ -0,0 +1,850 @@
...
+            <magazineSize>1</magazineSize>
+            <reloadTime>10</reloadTime>
+            <ammoSet>AmmoSet_164mmCannonBall_MortarSmall</ammoSet>
+          </AmmoUser>
+			
+          <FireModes>
+			<aiUseBurstMode>FALSE</aiUseBurstMode>
+			<aiAimMode>AimedShot</aiAimMode>
+			<noSnapshot>true</noSnapshot>
+			<noSingleShot>true</noSingleShot>
```
Comment (N7Huntsman) [inline Patches/Not Only Just A Cannon/Patch_Cannon_Turret.xml:415]: These guns should match the existing flak cannon in that they should only have single shot mode appear on the gizmo, as opposed to fully automatic like they do now.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

**Comment by Person243546** [comment]
The Dragonator has been patched now. (may be a bit op)

**Comment by N7Huntsman** [comment]
> The Dragonator has been patched now. (may be a bit op)

Well, it's currently only got a range of about 12. I think you mixed up range and minrange.

**Comment by Person243546** [comment]
It originally had a range of 2 so it was a close range one hit kill weapon. (I think they meant to make it a melee turret)

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**